test/functionalTest/cases/0000_large_requests/notification_different_sizes.test

```diff
@@ -241,7 +241,7 @@ Date: REGEX(.*)
 1
 ** Notification sizes:
  Sending message 1 to HTTP server: sending message of REGEX((19669|19670|19671|19672|19673|19674|19675)) bytes to HTTP server
- Sending message 2 to HTTP server: sending message of REGEX((21670|21671|21672|21673|21674|21675|21676)) bytes to HTTP server
+ Sending message 2 to HTTP server: sending message of REGEX((21669|21670|21671|21672|21673|21674|21675|21676)) bytes to HTTP server
```

Comment:
Probably this is not needed... use to be a glitch effect when develop evolves (in this case) from 1.0.0-next -> 1.1.0 -> 1.1.0-next.

Let's rollback this .test to its original status after merging develop now (note that this PR includes a develop merge, but not at the moment in which develop was prepared for the next cycle) and re-run the .test in order to check if my theory is correct.